feeds/wlan-ap/interAPcomm/Makefile

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libinterapcomm
+PKG_RELEASE:=1.0.0
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libinterapcomm
+  SECTION:=base
```

Comment:
these 2 fields appear to be missing
SECTION:=libs
CATEGORY:=Libraries

Reply:
It appears that the comments were not addressed in the latest version of the patch